### DISCUSSION TITLE
[DONE] Team Dashboard "disabled" count irritating?

### DISCUSSION THREAD

**Discussion by jone** [ParetoSecurity/pareto-mac#78] [General]
Our team dashboard says that I have disabled 7 checks, but I didn't disable any checks. When looking in the software part I realized, that the cause is that I do not have all supported software installed on my computer:
<img width="618" alt="Bildschirmfoto 2021-12-01 um 22 36 44" src="https://user-images.githubusercontent.com/7469/144318140-6b212ab0-58ec-46f8-a2bd-4e6dae4b4979.png">

The discussion is probably: what is the purpose of this counter?

I personally would like to see in the listing whether somebody of the company has disabled checks on purpose, so that I can quickly verify that nothing important was disabled. The amount of software that is not installed is not of interest for me here 😉 

By the way: I really like that software is checked for updates 👍 🎉

**Comment by zupo** [comment]
Gheesh, missed this in our testing, completely valid complaint!

**Comment by zupo** [comment]
These are now hidden in the UI. We'll probably revisit this in the future, to have a new state for such checks, but for now, the current solution works.
